csharp/build.sh

```diff
@@ -37,7 +37,7 @@ export CONFIGURATION=$STEP
 
 export STEP=$CONFIGURATION
 
-xbuild /p:Configuration=$CONFIGURATION $XBUILDOPT $PROJ
+xbuild /p:Configuration=$CONFIGURATION;AllowUnsafeBlocks=true $XBUILDOPT $PROJ
```

Comment:
@hebinhuang : just as the latest comment shown, 
quote the intact item as intact : "/p:Configuration=$CONFIGURATION;AllowUnsafeBlocks=true"